sklearn/cluster/optics_.py

```diff
@@ -88,17 +88,29 @@ class OPTICS(BaseEstimator, ClusterMixin):
 
     cluster_method : string, optional (default='dbscan')
         The extraction method used to extract clusters using the calculated
-        reachability and ordering. Possible values are "dbscan".
+        reachability and ordering. Possible values are "xi" and "dbscan".
 
     eps : float, optional (default=0.5)
         The maximum distance between two samples for them to be considered
-        as in the same neighborhood. Used ony when `cluster_method='dbscan'`.
+        as in the same neighborhood. Used ony when ``cluster_method='dbscan'``.
```

Comment:
s/ony/only/

I would not give a default value. The best default would likely be `max_eps`, wouldn't it?

Reply:
makes sense, done.